src/psyclone/gocean1p0.py

```diff
@@ -833,8 +876,11 @@ def gen_code(self, parent):
 
         # Generate the upper and lower loop bounds
         # TODO: Issue 440. upper/lower_bound should generate PSyIR
-        self.start_expr = Literal(self._lower_bound(), parent=self)
-        self.stop_expr = Literal(self._upper_bound(), parent=self)
+        self.start_expr = self._lower_bound(parent=self)
```

Comment:
As mentioned earlier, should the above TODO be removed, seeing as you now do generate PSyIR?

Reply:
Done.